ci/ansible/roles/pulp-backup/defaults/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# location of backup on remote
+fetch_to_localhost: true
+remote_bkup_dir: "{{ ansible_user_dir }}/remote_backups"
+local_bkup_dir/: "local_backups"
```

Comment:
Oh I was looking at the wrong line. That is absolutely a typo. Good eye. In fact, I imagine it sets a default variable to some value that is never used ( I imagine "{{ local_bkup_dir }}"  != "{{ local_bkup_dir/ }}"